galaxy-operator/bin/readyz.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import requests
+
+from requests.packages.urllib3.util.connection import HAS_IPV6
+
+
+def is_api_healthy(path):
+    """
+    Checks if API is healthy
+    """
+    address = "[::1]" if HAS_IPV6 else "127.0.0.1"
+    url = f"http://{address}:8000{path}"
+    print(f"Readiness probe: checking {url}")
+    response = requests.get(url, allow_redirects=True)
+    data = response.json()
+
+    if not data["database_connection"]["connected"]:
+        print("Readiness probe: database issue")
+        sys.exit(3)
+
+    if os.getenv("REDIS_SERVICE_HOST") and not data["redis_connection"]["connected"]:
+        print("Readiness probe: cache issue")
+        sys.exit(4)
+
+    print("Readiness probe: ready!")
+    sys.exit(0)
+
+
+def is_content_healthy(path):
+    """
+    Checks if Content is healthy
+    """
+    address = "[::1]" if HAS_IPV6 else "127.0.0.1"
+    url = f"http://{address}:24816{path}"
+    print(f"Readiness probe checking {url}")
+    response = requests.head(url)
+    response.raise_for_status()
+
+    print("Readiness probe: ready!")
+    sys.exit(0)
+
+
+"""
+PID 1 value (/proc/1/cmdline) may vary based on the gunicorn install method (RPM vs pip)
+The cmdline value for this PID looks like:
+```
+# pip installation
+gunicorn: master \[pulp-{content,api}\]
```

Comment:
The CI linter is just complaining about the backslashes here in the docstring.

```
Run [ ! -x .ci/scripts/extra_linting.sh ] || .ci/scripts/extra_linting.sh
./galaxy-operator/bin/readyz.py:51:18: W605 invalid escape sequence '\['
./galaxy-operator/bin/readyz.py:51:38: W605 invalid escape sequence '\]'
```

We can just remove them imo.